js/ui/grid_core/ui.grid_core.columns_view.js

```diff
@@ -734,6 +734,12 @@ exports.ColumnsView = modules.View.inherit(columnStateMixin).inherit({
         }
     },
 
+    getCellIndex: function($cell) {
```

Comment:
This method is overloaded in columnHeaders. Is this basic method necessary?

Reply:
It is necessary because I use getCellIndex in columnsView instead I should use it from the rowsView. It's not correct.